docs/DeveloperGuide.md

```diff
@@ -1,34 +1,283 @@
+
+This will require the implementation of more update methods in the ActivityList class to allow for the changes to be updated in the activityList after it has been edited. 
+
+#### 3.4.3 Design Considerations
+By letting the user edit the name and tags of the activity, it will allow them to correct any mistakes made during the data entry. This ensures that there is an accurate record of activities such as in cases where the user may be trying to record the same activity but has misspelled it, resulting in the program regarding it as a different activity where there would be multiple unnecessary new entries in the activity list, making the analysis of the time spent more tedious and inaccurate.
+
+However, by allowing the user to edit the start date and time, there may be potential inaccuracies in the actual activity recording. This is due to the fact that the time recorded in the program is based on the LocalDateTime. By introducing user input, the dates and time may be recorded incorrectly, defeating the purpose of the time tracking program. 
+
+### 3.5 Continue Feature
+The continue feature allows the user to continue a previously ended activity.
+
+#### 3.5.1 Current Implementation
+![Continue command sequence diagram](./pictures/continueActivity.png)
+
+**Continuing an activity:**
+
+When the user enters the command to continue an activity, a *ContinueCommand* object is created in *Parser*. The method *executeCommand()* of the *ContinueCommand* object is then called and does the following:
+
+ 1. Checks if the given activity name exists in the activityList by calling *findActivity()* (if it doesn’t an exception is thrown, omitted in the sequence diagram above)
+ 2. Gets the *name* and *tags* of the activity to be continued and saves it to a public static variable of *Parser* object
+ 3. Gets the current time and saves it to a public static variable of *Parser* object
+ 
+ ![End command sequence diagram](./pictures/endActivity.PNG)
+
+ **Ending a continued activity:**
+ 
+When the user wants to end the continued activity, an *EndCommand* object is created in *Parser.* The method *executeCommand()* of the *ContinueCommand* object is then called and it in turn executes the *saveActivity()* method of the *ActivityList* class. The continued activity is then saved by executing the following:
+
+ 1. Gets the current time and saves it to a public static variable of *Parser* object
+2.  Calculates the elapsed time using *between()* method of *Duration* class
+3.  Adds the elapsed time with the previous duration of the activity to get the *newDuration* using *plus()* method of Duration class
+4.  Calls the *updateDuration()* method, which updates the *duration* attribute of the continued activity in the *activityList* as well as the data.csv file
+
+#### 3.5.2 Design Considerations
+
+**Execution:**
+ - Continue by activity name (current implementation)
+ **Cons:** Activity names have to be unique.
```

Comment:
I feel that there is a minor formatting error here.  It will be clearer if you can break it into different lines instead of a single line. 
![image](https://user-images.githubusercontent.com/42601850/77987945-bc79db00-734d-11ea-91f8-f3113a52e37e.png)